core/scripts/cli/sponsor/withdraw.js

```diff
@@ -184,6 +200,9 @@ const withdraw = async (web3, artifacts, emp) => {
           tokensToWithdraw
         )} ${requiredCollateralSymbol} will process instantly`
       );
+      console.log(
+        `Hypothetical collateralization ratio if you choose to execute this withdrawal: ${newCollateralizationRatio}`
```

Comment:
this is super useful. when we get to building a Dapp for this having these kinds of calculations and the logic around them will make this process way quicker.